hackathon_site/dashboard/frontend/src/pages/Inventory/Inventory.js

```diff
@@ -1,26 +1,151 @@
 import React from "react";
-// import styles from "./Inventory.module.scss";
+import styles from "./Inventory.module.scss";
+import { useTheme } from "@material-ui/core/styles";
+import Drawer from "@material-ui/core/Drawer";
 import Header from "components/general/Header/Header";
 import Typography from "@material-ui/core/Typography";
+import TextField from "@material-ui/core/TextField";
+import useMediaQuery from "@material-ui/core/useMediaQuery";
+import Divider from "@material-ui/core/Divider";
+import IconButton from "@material-ui/core/IconButton";
+import Hidden from "@material-ui/core/Hidden";
+import Button from "@material-ui/core/Button";
+import RefreshIcon from "@material-ui/icons/Refresh";
+import SearchIcon from "@material-ui/icons/Search";
+import FilterListIcon from "@material-ui/icons/FilterList";
+import Grid from "@material-ui/core/Grid";
 import InventoryFilter from "components/inventory/InventoryFilter/InventoryFilter";
-import { inventoryCategories } from "testing/mockData";
+import Item from "components/inventory/Item/Item";
+import { inventoryCategories, inventoryItems } from "testing/mockData";
 
 const applyFilter = () => alert("Applies the filter");
 
 const removeFilter = () => alert("Removes all filters and resets form");
 
 const Inventory = () => {
+    const [mobileOpen, setMobileOpen] = React.useState(false);
+
+    const toggleFilter = () => {
+        setMobileOpen(!mobileOpen);
+    };
+
+    const mobileWidth = useMediaQuery(useTheme().breakpoints.up("md"));
+
     return (
-        <div>
+        <>
             <Header />
-            <Typography variant="h1">Hardware Inventory</Typography>
-            <p>IEEEEEE</p>
-            <InventoryFilter
-                categories={inventoryCategories}
-                applyFilter={applyFilter}
-                removeFilter={removeFilter}
-            />
-        </div>
+            <div className={styles.inventory}>
+                <Drawer
+                    className={styles.inventoryFilterDrawer}
+                    variant="permanent"
+                    variant={mobileWidth ? "permanent" : "temporary"}
+                    anchor="left"
+                    open={mobileOpen}
+                    onClose={toggleFilter}
+                >
+                    <InventoryFilter
+                        categories={inventoryCategories}
+                        applyFilter={applyFilter}
+                        removeFilter={removeFilter}
+                    />
+                </Drawer>
+                <div className={styles.inventoryBody}>
+                    <Typography variant="h1">Hardware Inventory</Typography>
+
+                    <div className={styles.inventoryBodyToolbar}>
+                        <Grid
```

Comment:
How come you're using a grid container here? Swapping this out for a div seems to work fine for me, all the flexbox magic is maintained